app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less

```diff
@@ -0,0 +1,171 @@
+@product-name-link__color: @color-elogic-gray-main;
+@product-name-link__color__active: @color-elogic-black;
+@product-name-link__color__hover: @color-elogic-black;
+@product-name-link__color__visited: @color-elogic-gray-main;
+
+@product-name-link__text-decoration: none;
+@product-name-link__text-decoration__active: @color-elogic-black;
+@product-name-link__text-decoration__hover: @color-elogic-black;
+@product-name-link__text-decoration__visited: @color-elogic-gray-main;
+
+@product-item__hover__background-color: @color-white;
+@product-item__hover__box-shadow: 3px 4px 4px 0 rgba(0, 0, 0, .3);
+
+@product-price__muted__color: @color-elogic-gray-discounts;
+
+//
+//  Common
+//  _____________________________________________
+
+& when (@media-common = true) {
+    .block-content {
+        .products-grid {
+            margin-top: 30px;
+        }
+    }
+
+    .product-items {
+        .lib-vendor-prefix-display(flex);
+        .lib-vendor-prefix-flex-wrap(wrap);
+        justify-content: space-between;
```

Comment:
just additional thing (`(justify-content)` mixin is missing in the `_utilities.less`):
create your own mixin with two parameters